src/Concordium.Sdk.csproj

```diff
@@ -45,29 +49,50 @@
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
     <PackageReference Include="System.Linq.Async" Version="6.0.1" />
+    <PackageReference Include="WebAssembly" Version="1.2.1" />
```

Comment:
Can you explain what this is for?

Reply:
We are using it to parse the wasm module and “entering” the custom sections to extract the schema.